locallib.php

```diff
@@ -15,89 +15,1146 @@
 // along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
 
 /**
- * Local functions.
+ * Local functions
```

Comment:
I'd suggest to remove this file completely and replace with a auto-loaded helper class or something.

Reply:
Ok. As we are porting as is we can ignore this